src/drafty.js

```diff
@@ -67,6 +67,8 @@ const ALLOWED_ENT_FIELDS = ['act', 'height', 'duration', 'incoming', 'mime', 'na
   'ref', 'size', 'state', 'url', 'val', 'width'
 ];
 
+const segmenter = new Intl.Segmenter();
```

Comment:
> This will fail in Firefox. Do you want to add dependency on https://www.npmjs.com/package/@formatjs/intl-segmenter?

Yes, you are right. But I thought `formatjs/intl-segmenter` dependency would go in webapp and polyfill will be enabled there too, instead of sdk.

> FormatJS can be useful elsewhere. There are some error messages in the package which are not localized. I would use FormatJS to localize them.

Can you please elaborate? I am unable to understand which errors you are referring to.

Reply:
1. Webapp imports `'@formatjs/intl'`, not `@formatjs/intl-segmenter/polyfill`. I think it's going to fail in FF without doing something like this: https://formatjs.io/docs/polyfills/intl-segmenter/
2. The SDK is currently not internationalized. It has quite a few hardcoded English strings. It would be nice to internationalize it too. I would use `FormatJS` for that.

Reply:
I have imported formatjs's `intl-segmenter` `polyfill` in `tinode-js` instead of `webapp`.

Certainly, sdk can be internationalized in another PR, but it is out of scope of this one.